README.md

```diff
@@ -118,6 +119,19 @@ interface NativeIOManager {
   // Renames the file from old name to new name atomically.
   [RaisesException] Promise<void> rename(DOMString old_name, 
                                          DOMString new_name);
+  
+  // Requests new capacity (in bytes) for usage by the current execution
+  // context.
+  [ CallWith = ScriptState, RaisesException ] Promise<unsigned long long>
+  requestCapacity(unsigned long long requested_capacity);
+  
+  // Releases unused capacity (in bytes) from the current execution context.
+  [ CallWith = ScriptState, RaisesException ] Promise<unsigned long long>
+  requestCapacity(unsigned long long requested_capacity);
```

Comment:
I think you meant release here

Reply:
Could you also expand the comment to mention what the return value is, both here and in requestCapacity?

Reply:
Done.